docs/data/material/getting-started/templates/checkout/Checkout.tsx

```diff
@@ -55,68 +117,297 @@ export default function Checkout() {
+                mode={mode}
+                toggleColorMode={toggleColorMode}
+                showCustomTheme={showCustomTheme}
+              />
+              <Stepper
+                id="desktop-stepper"
+                activeStep={activeStep}
+                sx={{
+                  width: '100%',
+                  height: '40px',
+                }}
+              >
+                {steps.map((label) => (
+                  <Step
+                    sx={{
+                      ':first-child': { pl: 0 },
+                      ':last-child': { pr: 0 },
+                    }}
+                    key={label}
+                  >
+                    <StepLabel>{label}</StepLabel>
+                  </Step>
+                ))}
+              </Stepper>
+            </Box>
+          </Box>
+          <Card
+            sx={{
+              display: { xs: 'flex', md: 'none' },
+              width: '100%',
+            }}
+          >
+            <CardContent
+              sx={{
+                display: 'flex',
+                width: '100%',
+                alignItems: 'center',
+                ':last-child': { pb: '16px' },
+              }}
+            >
+              <Box sx={{ flexGrow: 1 }}>
+                <Typography variant="subtitle2" gutterBottom>
+                  Selected products
+                </Typography>
+                <Typography variant="body1">
+                  <Typography variant="body1">
+                    {activeStep === 2 ? '144.97' : '$134.98'}
+                  </Typography>
+                </Typography>
+              </Box>
+              <InfoMobile />
+            </CardContent>
+          </Card>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              flexGrow: 1,
+              width: '100%',
+              maxWidth: { sm: '100%', md: 600 },
+              maxHeight: '720px',
+              gap: { xs: 5, md: 'none' },
+            }}
+          >
+            <Stepper
+              id="mobile-stepper"
+              activeStep={activeStep}
+              alternativeLabel
+              sx={{ display: { sm: 'flex', md: 'none' } }}
+            >
+              {steps.map((label) => (
+                <Step
+                  sx={{
+                    ':first-child': { pl: 0 },
+                    ':last-child': { pr: 0 },
+                  }}
+                  key={label}
+                >
+                  <StepLabel
+                    sx={{ '.MuiStepLabel-labelContainer': { maxWidth: '70px' } }}
+                  >
+                    {label}
+                  </StepLabel>
+                </Step>
+              ))}
+            </Stepper>
+            {activeStep === steps.length ? (
+              <Stack spacing={2}>
+                <Typography variant="h1" gutterBottom>
+                  📦
+                </Typography>
+                <Typography variant="h5" gutterBottom>
```

Comment:
```suggestion
                <Typography variant="h5">
```

Enough space on the parent component already